### PR TITLE
add display locations 

### PR DESCRIPTION
**Pull request by Smullz622**
add new display locations and update description for some of the existing locations

### CODE REVIEW

File: app/javascript/availability/libraries_locations.json [hunk 1]
```diff
@@ -138,22 +138,34 @@
...
-    "DISPLAY-BR": "Beaver - Display",
-    "DISPLAY-DN": "Dickinson Law (Carlisle) - Faculty Display",
-    "DISPLAY-DP": "Penn State Law (UP) - Faculty Display Case",
-    "DISPLAY-DS": "DuBois - Display",
-    "DISPLAY-ED": "Paterno - 5th Floor - Education Library Displays",
-    "DISPLAY-EG": "Engineering Library Display",
-    "DISPLAY-FE": "Fayette - New Titles or Featured Display Titles",
+    "DISPLAY-AA": "Altoona - On Display",
+    "DISPLAY-AB":	"Abington - On Display",
+    "DISPLAY-AC":	"Architcture - On Display",
```
Comment (jlandiseigsti) [inline app/javascript/availability/libraries_locations.json:143]: Should this be "Architecture"?

**Review by jlandiseigsti — COMMENTED** [review]
Think I might have found one spelling error, but otherwise looks ready.

File: app/javascript/availability/libraries_locations.json (hunk 1 above)
Comment (Smullz622) [reply app/javascript/availability/libraries_locations.json]: Ahh good catch. I fixed that along with the weird white space some of them have (didn't show up that way in vs code)

**Review by jlandiseigsti — APPROVED** [review]
LGTM!

**Pull request merged**